MaterialLib/MPL/MediumAuxiliaryFunction.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * File:   MediumAuxiliaryFunction.h
+ *
+ * Created on July 29, 2019, 2:48 PM
+ */
+
+#pragma once
+
+#include "MaterialLib/MPL/PropertyType.h"
+
+namespace MaterialPropertyLib
+{
+template <int GlobalDim>
+Eigen::Matrix<double, GlobalDim, GlobalDim> localTensor(
```

Comment:
Why 'local'? Is there a global tensor? I would prefer the name 'makeTensor'.

Reply:
or maybe `eigenTensor`...

Reply:
changed to `formTensor`.